### DISCUSSION TITLE
I have a letpos project in ssr mode, how can i integrated with TAURI? thanks

### DISCUSSION THREAD

**Discussion by skydig** [tauri-apps/tauri#9515] [Q&A]
as title

**Comment by FabianLars** [comment]
I guess you have 3 options:
- disable ssr (and of course rewrite your app to be able to run without it)
- host your frontend on a remote server and point tauri to that url
- include a localhost server in your tauri app, either as a sidecar (<https://tauri.app/v1/guides/building/sidecar>) or directly inline of the tauri app's rust source (idk if that works well with leptos)
